### ISSUE TITLE
Importer specification with individual cells

### ISSUE THREAD

**Issue by JustinasJas** [spine-tools/Spine-Toolbox#2836]
There is no way in Importer to map individual cells or column headings that are not on the top row.
The issue is the most notable for data sheets with metadata. Metadata at the top rows of the sheet prohibit use of column headings. Metadata itself can often be specified in one or two cells.
Currently working around this limitation by typing content of such cells manually as constants.

**Comment by soininen** [comment]
Did I understand this correctly, you would like to specify the exact coordinates (row, column) for a mapping?

**Comment by JustinasJas** [comment]
Yes, I think that would be the most general solution as it would allow to read content of any cell. I assume that is fitting for the toolbox as one can already type anything as a constant.
